kubernetes/config-maps/conductor/local_settings.py.j2

```diff
@@ -0,0 +1,37 @@
+from ..production import *
+
+from .email_settings import *
+from .timing_settings import *
+
+# Create secrets.py and put passwords and private keys in it.
+# Don't commit it into the repository.
+from ..secrets.secrets import (
+    STORAGE_SECRET_KEY,
+    STORAGE_DATABASE_PASSWORD,
+    EMAIL_PASSWORD,
+    CONCENT_PUBLIC_KEY,
+    SENTRY_DATA_SOURCE_NAME,
+)
+
+SECRET_KEY = STORAGE_SECRET_KEY
+
+DATABASES['default']['NAME']     = '{{ storage_database_name }}'
+DATABASES['default']['USER']     = '{{ storage_database_user }}'
+DATABASES['default']['PASSWORD'] = STORAGE_DATABASE_PASSWORD
+DATABASES['default']['HOST']     = '{{ storage_database_host }}'
+DATABASES['default']['PORT']     = '5432'
+
+ALLOWED_HOSTS = [
+    "conductor.default.svc.cluster.local",
```

Comment:
Please add `conductor` and `conductor.default` here.

Reply:
Done.